src/Navigation/RootContainer.tsx

```diff
@@ -2,11 +2,11 @@ import * as React from "react";
 import { NavigationContainer } from "@react-navigation/native";
 import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
 import { BottomTabParamList } from "./NavigationTypes";
-import HomeScreen from "../Pages/HomeScreen";
-import CategoriesScreen from "../Pages/CategoriesScreen";
-import SearchScreen from "../Pages/SearchScreen";
-import UserRecipesScreen from "../Pages/UserRecipesScreen";
-import ProfileScreen from "../Pages/ProfileScreen";
+import HomeScreen from "../pages/HomeScreen";
+import CategoriesScreen from "../pages/CategoriesScreen";
+import SearchScreen from "../pages/SearchScreen";
```

Comment:
you should stick to 1 way. you can name the folder with an uppercase letter or a lowercase letter, not both

Reply:
All folders were converted to be lowercase written. Did I miss something?